meetup-organizing.md

```diff
@@ -0,0 +1,38 @@
+# Organizing a meetup
+
+The following details the basic process for working with the Quorum to plan a meetup, confirm a speaker and topic, and provide any needed information or materials for the meetup event.
+
+The suggested timeframe for completing each of the preparation steps are:
+
+|   Step    |   Timeframe    |
+|  ---  |  ---  |  ---  |  ---  |
+|   Confirm details    |   1 month before    |
+|   Create event    |   3 weeks before    |
+|   Publicize event    |  2 weeks before     |
+
+## Confirm details
+
+When you have a speaker lined up and confirmed contact Alyssa Rock via [Slack](https://join.slack.com/t/writethedocs/shared_invite/zt-lqlaz4w6-WCnSTVD25MebjiqowO8EvQ) message (@Alyssa Rock) or [email thread](mailto:alyssa.rock@gmail.com). Together you can discuss the speaker and topic, determine whether it's a good fit for a Quorum meetup event, and finalize details, including:
+
+*  Date and time for the presentation
+*  Presentation title and description
+*  Obtaining a brief speaker bio
+*  Any other concerns, materials, or information
+
+## Create event
+
+Once the details have been nailed down, Alyssa will create the Zoom event and Meetup event.
+
+## Publicize event
+
+Once the Zoom and Meetup events are created, we will announce it to the rest of the Quorum meetup group, and remind Quorum members to join the meetup and help publicize it to their individual meetup groups members.
+
+Great ways to publicize the Quorum meetups in your local meetup groups include:
+
+*  Adding a description and meetup event link to Slack in your local meetup channel.
+*  Sending out an email via Meetup.org to your group with the meetup event info.
+*  Using Twitter or other social media avenues to further promote the meetup event.
+
```

Comment:
**Suggestion (non-blocking):** Before the Resources section, maybe we could also have a section for "Practice emceeing" or "Dry-run through" or something like that. In this section, we could mention that about 1 week before the event, the meetup organizer can optionally meet with me to do a dry-run of the event. It takes about 30 minutes and we just talk through all the logistics and decide which ice-breaker questions we want to ask.

I've done this with the past two events we've held and it's gone really well. Andrea also invited her panel of speakers to come to the dry-run so that we could answer their questions and I think it really put them at ease.

At a minimum, you might want to open a 3-way email thread or Slack conversation with me, you, and the speaker so that the communication channels are open for sharing pertinent details.